Compile error

Hi there.

This is failing to build for me, with error:

```
/fwbuild-volumes/1.26/apps/mos-esp32-oled/esp32/build_contexts/build_ctx_508263544/src/main.cpp: In function 'void* atob(unsigned char*, int)':
/fwbuild-volumes/1.26/apps/mos-esp32-oled/esp32/build_contexts/build_ctx_508263544/src/main.cpp:12:59: error: 'cs_base64_decode' was not declared in this scope
   cs_base64_decode(data, inputLength, (char *)result, NULL);
```

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -1,3 +1,4 @@
+#include "mgos.h"
 #include "mgos_arduino_ssd1306.h"
 
 #ifdef __cplusplus

